src/main/kotlin/com/noumenadigital/npl/cli/NplCommandExecutor.kt

```diff
@@ -0,0 +1,34 @@
+package com.noumenadigital.npl.cli
+
+import com.noumenadigital.npl.cli.exception.NplCliException
+import com.noumenadigital.npl.cli.service.NplCommandsParser
+import java.io.Writer
+
+class NplCommandExecutor(
```

Comment:
I'm confused by the naming. We have two different `NplCommandExecutor` but one is a class and one is an interface, and they're not really related?